components/webpack/prod.config.js

```diff
@@ -33,7 +39,10 @@ module.exports = {
     rules: [
       {
         test: /\.tsx?$/,
-        loader: 'awesome-typescript-loader'
+        loader: 'awesome-typescript-loader',
+        query: {
+          useBabel: true
```

Comment:
do we need babel here?

Reply:
If not pls PR to remove, but I'll merge this to unblock devops. Thanks.

Reply:
added https://github.com/brave/brave-core/pull/152